src/cljs/main/broadfcui/page/workspace/workspace_common.cljs

```diff
@@ -47,6 +48,23 @@
 (defn config->id [config]
   (select-keys config [:namespace :name]))
 
+(defn- referenced-method-parts [config]
+  ((juxt :sourceRepo :methodNamespace :methodName :methodPath :methodVersion) (:methodRepoMethod config)))
```

Comment:
This strikes me as a little weird. Maybe:
```clojure
(replace (:methodRepoMethod config) [:sourceRepo :methodNamespace :methodName :methodPath :methodVersion])
```

Reply:
thanks for this - the `juxt` was cut-paste from the previous table definition and I also found it weird but didn't investigate anything better - I like your suggestion a lot more!